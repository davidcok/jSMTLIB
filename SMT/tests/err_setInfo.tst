; not allowed to set pre-defined info
(set-info :name "x")
(set-info :authors "x")
(set-info :version "0")
(set-info :error-behavior x)
(set-info :all-statistics x)
(set-info :reason-unknown x)
