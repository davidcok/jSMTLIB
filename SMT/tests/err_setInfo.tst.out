(set-info :name "x")
          ^^^^^
(error "Setting the value of a pre-defined keyword is not permitted: :name")
(set-info :authors "x")
          ^^^^^^^^
(error "Setting the value of a pre-defined keyword is not permitted: :authors")
(set-info :version "0")
          ^^^^^^^^
(error "Setting the value of a pre-defined keyword is not permitted: :version")
(set-info :error-behavior x)
          ^^^^^^^^^^^^^^^
(error "Setting the value of a pre-defined keyword is not permitted: :error-behavior")
(set-info :all-statistics x)
          ^^^^^^^^^^^^^^^
(error "Setting the value of a pre-defined keyword is not permitted: :all-statistics")
(set-info :reason-unknown x)
          ^^^^^^^^^^^^^^^
(error "Setting the value of a pre-defined keyword is not permitted: :reason-unknown")
